sqlest/src/main/scala/sqlest/executor/Database.scala

```diff
@@ -224,6 +229,38 @@ case class Transaction(database: Database) extends Session(database) {
       }
     }
 
+  def executeCommandReturningKeys[T](command: Command)(implicit columnType: ColumnType[T]): RowCountAndKeys[T] =
+    withConnection { connection =>
+      val (preprocessedCommand, sql, argumentLists) = database.statementBuilder(command)
+      val startTime = new DateTime
+      try {
+        val preparedStatement = prepareStatement(connection, preprocessedCommand, sql, argumentLists)
+        try {
+          val keys = List[T]()
+          val result = preparedStatement.executeUpdate
+          val rs = preparedStatement.getGeneratedKeys
```

Comment:
Not according to the spec.  Happy to change.

https://docs.oracle.com/javase/7/docs/api/java/sql/Statement.html#close()